src/main/dissolve.h

```diff
@@ -109,26 +109,10 @@ class Dissolve : public Serialisable<>
      * Layers
      */
```

Comment:
This needs renaming now:
```suggestion
     * Processing Module Data
     */
```